x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiFlexGroup, EuiText, EuiFlexItem } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { CustomSeriesColorsMap, DataSeriesColorsValues, getSpecId } from '@elastic/charts';
+
+const FlexGroup = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+export const ChartHolder = () => (
+  <FlexGroup justifyContent="center" alignItems="center">
+    <EuiFlexItem grow={false}>
+      <EuiText size="s" textAlign="center" color="subdued">
+        Chart Data Not Available
```

Comment:
You will need a i18n key here for `Chart Data Not Available`. It doesn't have to be a new separate file if you don't want it to be. Some people prefer to use i18n inline within their files and I think that's fine.